mflowgen/glb_top/custom-init/outputs/floorplan.tcl

```diff
@@ -22,22 +22,22 @@ set core_margin_l [expr 5 * $horiz_pitch]
 
 # Margins between glb tiles and core edge
 set tile_margin_t [expr 100 * $vert_pitch]
-set tile_margin_b [expr 50 * $vert_pitch]
-set tile_margin_l [expr 500 * $horiz_pitch]
+set tile_margin_b [expr 100 * $vert_pitch]
+set tile_margin_l [expr 350 * $horiz_pitch]
 set tile_margin_r [expr 50 * $horiz_pitch]
 
 set tiles [get_cells *glb_tile*]
 set tile_width [dbGet [dbGet -p top.insts.name *glb_tile* -i 0].cell.size_x]
-# set tile_gap [expr 20 * $horiz_pitch]
 set tile_gap 0
+set num_tiles_for_gap 4
```

Comment:
What does this parameter mean? A comment here to explain what's going on would be helpful.

Reply:
It looks like you only add the gap every `num_tiles_for_gap` tiles, right?

Reply:
yep. I will add comments